src/query/sql/src/planner/optimizer/heuristic/decorrelate.rs

```diff
@@ -342,16 +342,16 @@ impl SubqueryRewriter {
                     )
                 };
                 let mark_join = LogicalInnerJoin {
-                    left_conditions,
-                    right_conditions,
+                    left_conditions: right_conditions,
```

Comment:
Maybe we can give a little different additional cost for different Join?

Reply:
Do you mean join cardinality in join order?

Reply:
for example, let left mark join cost a little more than right mark join.